intellij/src/saros/intellij/ui/views/buttons/ConsistencyButton.java

```diff
@@ -149,11 +151,11 @@ private void setInconsistent(boolean isInconsistent) {
     sessionInconsistencyState.isInconsistent = isInconsistent;
 
     if (isInconsistent) {
-      setEnabledFromUIThread(true);
+      setEnabled(true);
```

Comment:
I do not know the IntelliJ/AWT/Swing guidelines but it is pretty weird that this call is explicitly synchronized with the UI Thread but setButtonIcon and setToolTipText is not (I do not know the internal implementation but your setEnabled override just calls the original setEnabled method. However you did not do that for setButtonIcon and setToolTipText).

Reply:
As mentioned in the comment on the actual implementation, I just assumed that the pre-existing method to synchronize the enable access was put there on purpose/for a reason. It can probably be removed.